Vungle/src/main/java/com/mopub/mobileads/VungleAdapterConfiguration.java

```diff
@@ -42,7 +42,7 @@ public String getAdapterVersion() {
     @Nullable
     @Override
     public String getBiddingToken(@NonNull Context context) {
-        return null;
+        return Vungle.availableBidTokens();
```

Comment:
Could we pass the limit here instead of hardcoding it on sdk side?

Reply:
ok